### ISSUE TITLE
`source` being appended to rtd builds?

### ISSUE THREAD

**Issue by dylanh724** [readthedocs/readthedocs.org#11384]
## Details

My dir tree:

- source/
   - conf.py
   - index.rst
   - content/
      - foo.rst

To access foo.rst locally in a toctree, it's `content/foo`

To access foo.rst **in your hosted RST build**, it's `source/content/foo` **(as if there's an extra source dir)**.

## Expected Result

Local and RTD should map almost 1:1; I'm not sure what's going on with the "extra" source dir. 

## Actual Result

Eg:

- source/
   - conf.py
   - index.rst
   **- source/**
         - content/
            - foo.rst

For som reason, RST adds the extra **source** dir. I can either have successful local builds or rtd builds, but not both.

**Comment by dylanh724** [comment]
What's odd is that I had a demo of it working in both local and rtd 1:1 -- I tried reverting back and it oddly didn't work (same bug). It's as if RTD just updated and inadvertently appended a source dir. I really want it to be me, but how can I possibly inadvertently add an extra source dir that *only* shows in rtd and not local?

**Comment by dylanh724** [comment]
![image](https://github.com/readthedocs/readthedocs.org/assets/8840024/1b2d036b-bb29-47eb-81a7-114cbf91f329)

**Comment by stsewd** [comment]
Hi, please provide the URL of your project on RTD so we can help you.

**Comment by dylanh724** [comment]
> Hi, please provide the URL of your project on RTD so we can help you.

Ah, I don't think I'm allowed to, yet. Does this mean `checkouts/latest/source/source` with the double `source` is unique to my project, somehow?

**Comment by dylanh724** [comment]
There really was an update on June 4th!

https://docs.readthedocs.io/en/stable/changelog.html

**Comment by stsewd** [comment]
> Ah, I don't think I'm allowed to, yet. Does this mean checkouts/latest/source/source with the double source is unique to my project, somehow?

Note that projects on readthedocs.org are public, anyone can see them. We haven't received any other reports related to this, without a build to look at, there isn't much we can do to help or debug this.

> Mine last worked late May 27: Seems there were two patches since then. Hmm.

Did a quick look, and don't see anything relevant that may have affected that.

**Comment by dylanh724** [comment]
@stsewd https://docs.readthedocs.io/en/stable/changelog.html#version-11-8-1

This update fixed the version (had to revert my workarounds).

Whatever was between the `June 4th, 28th` that caused the issue is now fixed in the `Oct 1` update. 

Overnight, new build content wasn't showing that would build fine locally with only aesthetic content changes. I saw there was a patch and tested reverting my bandaid and... it now works! _There really was something_ that was causing a `source/source` path
